src/components/timeline/form/TimelineEditorLayerFilter.svelte

```diff
@@ -38,8 +40,11 @@
   $: if (filterString) {
     const filterStringLower = filterString.toLocaleLowerCase();
     filteredValues = options.filter(item => item.toLocaleLowerCase().indexOf(filterStringLower) > -1);
+    // Unique
```

Comment:
Can you comment a little more on why this is here?

Reply:
If you're just referring to the comment on line 43, I think its an artifact of a longer comment that got squished to the in-line `// uniqueness` one!
And if you're referring to the unique-ness itself, `filterString` is meant to track the current filter for external event types on the timeline (there's a similar implementation in `ExternalSourceManager.svelte` also). I believe the unique'ing here is just extra protection over getting too many occurrences of a single value in the filter, but correct me if I'm wrong @pranav-super.

Reply:
Yeah, I can confirm that the line that the now-inlined `// Unique` comment referred to is for the purpose of ensuring only unique, non-duplicate entries exist in `filteredValues`, as that was an issue at the time of writing this line. @duranb @AaronPlave should a more verbose comment be included or is this sufficient?